bindings/rust/src/bindings.rs

```diff
@@ -212,41 +213,48 @@ extern "C" {
 
     pub fn free_trusted_setup(s: *mut KZGSettings);
 
+    pub fn blob_to_kzg_commitment(
```

Comment:
Just out of curiosity: Did you do these changes to `bindings.rs` by running `bindgen`?

Reply:
No, I did not. Updated these manually.

Reply:
@pawanjay176 Hey hey. Have you considered integrating bindgen to the build.rs? https://rust-lang.github.io/rust-bindgen/tutorial-3.html

Reply:
I used bindgen to generate the first pass of the bindings. It's a bit of a pain to keep it automated as the auto generated code needs a few tweaks to get working even after using all available config options in the bindgen builder. 
Also, I figured if we end up changing the interface, it will need more work to get stuff compiling and all tests and benchmarks running again.

Reply:
> I used bindgen to generate the first pass of the bindings. It's a bit of a pain to keep it automated as the auto generated code needs a few tweaks to get working even after using all available config options in the bindgen builder. Also, I figured if we end up changing the interface, it will need more work to get stuff compiling and all tests and benchmarks running again.

Hmm, the interface has changed since the first pass of the bindings and there was also a new release of bindgen two weeks ago. Would you be up for running bindgen once more for the latest interface (which is most likely gonna be the final one) so that we have the latest and greatest?

Reply:
> Sorry I'm a bit confused. If I understood you correctly earlier, you were suggesting that we auto-generate the bindings in `build.rs` instead of manually changing bindings.rs everytime there's a change in the C interface.

Sorry for the confusion. Auto-generating was indeed what I suggested in the beginning, but after you explained that it's complicated to bake bindgen into `build.rs`, I changed my mind and just suggested to run the latest bindgen manually one more time.

I suggested that because it's been many months since that original bindgen run and the interface has changed quite a bit since then. Also there has been a new bindgen version since then and the [bindgen docs](https://rust-lang.github.io/rust-bindgen/tutorial-1.html)  say "Please always use the latest version of bindgen, it has the most fixes and best compatibility".